tests/unit/bokeh/test_model.py

```diff
@@ -134,6 +134,52 @@ def test_all_builtin_models_default_constructible() -> None:
             bad.append(name)
         assert bad == []
 
+def test_select() -> None:
+        # we aren't trying to replace test_query here, only test
+        # our wrappers around it, so no need to try every kind of
+        # query
+        root1 = SomeModel(a=42, name='a')
+        root2 = SomeModel(a=43, name='c')
+        root3 = SomeModel(a=44, name='d')
+
+        # select()
+        assert set([root1]) == set(root1.select(dict(a=42)))
+        assert set([root1]) == set(root1.select(dict(name='a')))
+        assert set([root2])  == set(root2.select(dict(name='c')))
+        assert set()  == set(root1.select(dict(name='nope')))
+
+        # select() on object
+        assert set() == set(root3.select(dict(name='a')))
+        assert set([root3]) == set(root3.select(dict(a=44)))
+
+        # select_one()
+        assert root3 == root3.select_one(dict(name='d'))
+        assert None == root1.select_one(dict(name='nope'))
+        got_error = False
+        try:
+            root1.select_one(dict(name='c'))
+        except ValueError as e:
+            got_error = True
+            assert 'Found more than one' in repr(e)
+        assert got_error == False
```

Comment:
Use `pytest.raises(...)`.

Reply:
Okay. Would you like me to change it in `test_document.py` also?
Its the same in there.

Reply:
@mattpap I don't think there is a possibility of more than one select in a single object.

Reply:
These doc tests are old, it would be *nice* but not required to update them to use `pytest.raises` (but definitely should use `pytest.raises` on new code)